src/styles/components/DataGridWithInfinteScroll.scss

```diff
@@ -0,0 +1,755 @@
+    padding: unset;
+    margin: unset;
+
+    table {
+      margin-top: unset;
+    }
+  }
+
+  .card-title {
+    margin-bottom: 1.19rem;
+  }
+
+  .medium-button {
+    margin: unset;
+    margin-top: 1.1975rem;
+    height: 1.6rem !important;
+  }
+}
+
+.empty-datagrid {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 70vh;
+  font-size: 16px;
+  position: relative;
+}
+
+.empty-table-border {
+  border-top: 1px solid gray;
+}
+
+.resizer {
+  display: inline-block;
+  background: #e4e4e4;
+  width: 1px;
+  height: 24px;
+  position: absolute;
+  right: 0;
+  top: 0;
+  transform: translateX(50%);
+  z-index: 1;
+  touch-action: none;
+  color: #e4e4e4;
+  margin-top: 0.5rem;
+
+  &.isResizing {
+    background: #e4e4e4;
+  }
+}
+
+.checkbox {
+  margin: 0.5em 0 0.5em 0.5em;
+}
+
+.checkbox .tag {
+  color: #595959;
+  display: block;
+  float: left;
+  font-weight: bold;
+  position: relative;
+  width: 120px;
+}
+
+.checkbox label {
+  display: inline;
+}
+
+.checkbox .input-assumptive {
+  display: none;
+}
+
+.input-assumptive+label:after {
+  background-color: #fafafa;
+  border: 1px solid #cacece;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05), inset 0px -15px 10px -12px rgba(0, 0, 0, 0.05);
+  display: inline-block;
+  transition-duration: 0.3s;
+  width: 16px;
+  height: 16px;
+  content: "";
+  margin-left: 0.625rem;
+  opacity: 0;
+}
+
+.input-assumptive:checked+label:after {
+  background-color: #f5f3f3;
+  border-color: #e4e4e4;
+  opacity: 0;
+}
+
+.reset-button {
+  color: $blue-600;
+}
+
+.filter-icon {
+  float: right;
+  height: 1.5rem;
+  width: 1.6rem;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+
+.filter-open {
+  float: right;
+  height: 1.1rem;
+  width: 1.1rem;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+
+.expansion-column {
+  padding: 0 1rem;
+}
+
+.expansion-icon {
+  height: 14px;
+  width: 7.82px;
+  display: flex;
+
+  img {
+    height: 100%;
+    width: 100%;
+  }
+}
+
+.expanded {
+  transform: rotate(90deg);
+}
+
+.align-close-icon {
+  position: absolute;
+  right: 0;
+  margin: 0.625rem;
+}
+
+.ColumnSelect {
+  padding: 0.625rem;
+  width: max-content;
+}
+
+.ColumnSelect-header {
+  font-weight: 700 !important;
+}
+
+.ColumnSelect-reset-button {
+  padding-bottom: 0 !important;
+  padding-top: 0 !important;
+  padding-right: 0.031rem;
+  padding-left: 0.031rem;
+  height: 40px;
+  border: 1px solid lightgray;
+  width: 120px !important;
+}
+
+.Reset-button-margin-css {
+  margin: 0 !important;
+}
+
+.ColumnSelect-header-names {
+  padding-left: 4px;
+}
+
+.column-switch {
+  border-radius: var(--clr-global-borderradius, 0.15rem);
+  padding: 0.9rem;
+  background-color: var(--clr-datagrid-popover-bg-color, white);
+  border-width: var(--clr-global-borderwidth, 0.05rem);
+  border-style: solid;
+  border-color: var(--clr-datagrid-popover-border-color, #cccccc);
```

Comment:
Please don't use hardcoded colors. use __Colour.SCSS file. It already have all DD colors

Reply:
Done